code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java

```diff
@@ -368,6 +369,21 @@ protected HandlerResult handleStartError(ProcessState state, Instance instance,
         throw e;
     }
 
+    protected int incrementDepTry(ProcessState state) {
+        DataAccessor accessor = DataAccessor.fromMap(state.getData()).withScope(InstanceStart.class).withKey("depTry");
+
+        Integer computeTry = accessor.as(Integer.class);
+        if (computeTry == null) {
+            computeTry = 0;
+        }
+
+        computeTry++;
+
+        accessor.set(computeTry);
```

Comment:
Yes, because the process state always gets persisted unless the server crash, at point we don't really care if we lost one count.